microshift_release_notes/microshift-4-16-release-notes.adoc

```diff
@@ -63,6 +63,10 @@ With this release, you can configure a custom server certificate that has been i
 
 You can now configure audit logging policies to manage the retention policies for log files, ensuring that edge devices with limited storage capacities are not hampered by accumulated logging data. To configure audit log policies, use settings such as a maximum file size limit and maximum retained files to set a limit on log storage size. You can also choose an audit policy profile to specify the data collected. See xref:../microshift_configuring/microshift-audit-logs-config.adoc#microshift-audit-logs-config[Configuring audit logs].
 
+[id="microshift-4-16-certificates-cleaning"]
+==== Cleaning up the certificates are supported
+With this release, you can clean up the custom certificates. See xref:../microshift_configuring/microshift-custom-ca.adoc#microshift-custom-ca-certificates-cleaning_microshift-custom-ca[Cleaning up and recreating the custom certificates] for details.
```

Comment:
We typically recommend the following standard format of "For more information, see <link>"

```suggestion
With this release, you can clean up custom certificates. For more information, see xref:../microshift_configuring/microshift-custom-ca.adoc#microshift-custom-ca-certificates-cleaning_microshift-custom-ca[Cleaning up and recreating the custom certificates].
```

Reply:
done